### PR TITLE
Add doc on how to merge a PR from github

### PR DESCRIPTION
**Pull request by mscherer**

### CODE REVIEW

File: tools/ansible.rst [hunk 1]
```diff
@@ -52,3 +52,20 @@ To push for a change, start by cloning the repository::
 Then modify and push to the same repository. If you are in the group `admins`,
 then you will be able to push.  If not, then you can send the patch on
 gluster-infra mailling list.
+
+Fetching a PR from github
+-------------------------
+
+If anyone open a PR on github, you can merge it (after proper review) using the
```
Comment (nigelbabu) [inline tools/ansible.rst]: *opens

File: tools/ansible.rst [hunk 2]
```diff
@@ -52,3 +52,20 @@ To push for a change, start by cloning the repository::
...
+following process, for the PR number 4. I will assume that the repository is setup
+like this with 2 remote, 1 to fetch and 1 to push::
```
Comment (nigelbabu) [inline tools/ansible.rst]: remotes

File: tools/ansible.rst [hunk 3]
```diff
@@ -52,3 +52,20 @@ To push for a change, start by cloning the repository::
...
+
+    git add remote bastion ssh://salt-master.gluster.org/srv/git_repos/public
```
Comment (nigelbabu) [inline tools/ansible.rst]: You might want to make this a `git clone`

**Review by nigelbabu — APPROVED** [review]
Looks good. Some grammar and style changes that'd be good to have.

File: tools/ansible.rst (hunk 3 above)
Comment (mscherer) [reply tools/ansible.rst]: I could maybe merge that with the previous section, and use a trick with push url.

**Pull request merged**